modules/sdk-lib-mpc/src/tss/ecdsa/noSmallFactorsProof.ts

```diff
@@ -12,20 +12,47 @@ const ORDER = BigInt('0xfffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8c
 const ELL = BigInt(256);
 const EPSILON = BigInt(BigInt(2) * ELL);
 
-/**
- * Generate pseudo-random challenge value $e$ for $(N, w)$.
- * @param N - the prime number to verify is a product of two large primes.
- * @param w - a random number with the same bitLength as N, that satisfies the Jacobi of w is -1 wrt N.
- * @returns {bigint} - challenge value $e$.
- */
-function generateE(N: bigint, w: bigint): bigint {
-  const digest = createHash('shake256', { outputLength: 1 + Math.floor((bitLength(ORDER) + 7) / 8) })
+function hash(N: bigint, w: bigint, nonce: Buffer): Buffer {
+  // NOTE: there's a bug in node type file for crypto that prevents us from using Hash.copy({ outputLength: ... })
```

Comment:
Is this still the case? Node types where updated recently

Reply:
Checked https://github.com/DefinitelyTyped/DefinitelyTyped for its latest for v16 and v18, both still have the bug.